scripts/submit-pr.sh

```
@@ -41,6 +41,68 @@ if [[ -z "$MODE" ]]; then
+get_coderabbit_feedback() {
+    local pr_number=$1
+
+    # Get inline review comments from CodeRabbit
+    echo "## Inline Comments"
+    gh api "repos/${REPO}/pulls/${pr_number}/comments" \
+        --jq '.[] | select(.user.login | startswith("coderabbitai")) | "- \(.path):\(.line // .original_line // "?") \(.body | split("\n")[0] | gsub("^_⚠️ Potential issue_ \\| _🟡 Minor_"; "[MINOR]") | gsub("^_⚠️ Potential issue_ \\| _🔴 Major_"; "[MAJOR]"))"' 2>/dev/null || echo "  (none)"
+
+    echo ""
+
+    # Get review body and extract nitpicks
+    local review_body=$(gh api "repos/${REPO}/pulls/${pr_number}/reviews" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)